src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -105,31 +105,30 @@ We can overcome this limitation by using non-standard ports like 4501/UDP and 50
 Additionally, the default OpenShift deployments do not allow assigning an elastic public IP
 to existing worker nodes, something that's necessary at least on one end of the IPsec connections.
 
-To handle these requirements on AWS, we provide a script that will prepare your AWS OpenShift deployment for Submariner, and will create an
-additional gateway node with an external IP.
+To handle these requirements on AWS, we provide a script
+[prep_for_subm.sh](https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh),
+that will prepare your AWS OpenShift deployment for Submariner, and will create an additional gateway node with an external IP:
 
 ```bash
 
 curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
 chmod a+x ./prep_for_subm.sh
 
-./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+./prep_for_subm.sh <ocp-install-path>      # respond yes when terraform asks to approve, or add after path: -auto-approve
 
 ```
 
-> **_INFO_** Please note that  **oc**, **aws-cli**, **terraform**, and **unzip** need to be installed before running the `prep_for_subm.sh` script.
+> **_INFO_** Please note that  **oc**, **aws-cli**, **terraform**, and **wget** need to be installed before running the `prep_for_subm.sh` script.
 
-Currently, prep_for_subm.sh script does not support specifying custom IPsec ports.
-Until the [issue](https://github.com/submariner-io/submariner/issues/240) is resolved, execute the following commands to open the necessary ports:
+It also supports specifying custom IPsec ports, and instance type for the gateway, by setting enviroment variables in advance. For example:
```

Comment:
You can remove this as this is a Quickstart guide and the issue is now fixed.

Reply:
Why shouldn't we mention that custom ports are also optional ?

Reply:
For this particular quickStart guide, we have to use custom ports and not the default ports. So, my suggestion is to remove the details associated with what prep_for_subm.sh will be supporting and just give instructions on how to configure AWS with custom ports.